snapcraft/commands/daemon.start

```diff
@@ -180,6 +180,12 @@ echo "==> Preparing /run/bin"
 mkdir -p "/run/bin"
 export PATH="/run/bin:${PATH}"
 
+if [ "${qemu_external_snap:-"false"}" = "true" ]; then
+    echo "==> Setting up external QEMU snap integration"
+    export LD_LIBRARY_PATH="${SNAP_CURRENT}/opt/external-qemu/lib/${ARCH}:${SNAP_CURRENT}/opt/external-qemu/lib/${ARCH}/pulseaudio:${SNAP_CURRENT}/opt/external-qemu/lib/${ARCH}/ceph:${LD_LIBRARY_PATH}"
```

Comment:
I see pulseaudio in there and I wonder about pipewire?

Reply:
it's because QEMU was failing on attempt to load necessary libs from `${SNAP_CURRENT}/opt/external-qemu/lib/${ARCH}/pulseaudio`. Do you think we should add `${SNAP_CURRENT}/opt/external-qemu/lib/x86_64-linux-gnu/pipewire-0.3/` too?

Reply:
I don't know pipewire at all, but maybe? If so, could we use a shell glob to avoid hardcoding the version in the dir name?

Reply:
>I don't know pipewire at all, but maybe?

Unfortunately, I don't know too ;-)

>If so, could we use a shell glob to avoid hardcoding the version in the dir name?

Hm, something like:
```
LD_LPATH_PIPEWIRE="$(readlink -f "${SNAP_CURRENT}"/opt/external-qemu/lib/"${ARCH}"/pipewire-*/)"
export LD_LIBRARY_PATH="${SNAP_CURRENT}/opt/external-qemu/lib/${ARCH}:${SNAP_CURRENT}/opt/external-qemu/lib/${ARCH}/pulseaudio:${SNAP_CURRENT}/opt/external-qemu/lib/${ARCH}/ceph:${LD_LPATH_PIPEWIRE:+${LD_LPATH_PIPEWIRE}:}${LD_LIBRARY_PATH}"
```

?

Reply:
Yeah, that `readlink` works.

Reply:
Have fixed!